app-creator/client/src/data/templates.ts

```diff
@@ -0,0 +1,23 @@
+import { TemplateItem } from '../client/fetch-templates';
+
+class TemplatesManager {
+  private templates: TemplateItem[];
+
+  constructor() {
+    this.templates = [];
```

Comment:
This may not be needed, if you declare `private templates: TemplateItem[] = [];` at the top?